test/unit/lodash.spec.js

```diff
@@ -0,0 +1,727 @@
+                expect(result).to.eql([15, 30, 37]);
+            });
+        });
+
+        describe('propertyOf2', function () {
+            var params,
+                elem;
+            beforeEach(function(){
+                params = datasets.propertyOf1();
+                elem = params[0];
+            });
+            it('should check type', function(){
+                expect(elem instanceof Object).to.eql(true);
+            });
+            it('should value of properties in obj', function(){
+                var obj = params[0];
+                expect(obj).to.have.property('number1');
+                expect(obj.number1).to.eql([ 5, 12, 15, 30, 37, 48 ]);
+                expect(obj).to.have.property('number2');
+                expect(obj.number2).to.eql([ 5, 12, 15, 30, 37, 48 ]);
+                expect(obj).to.have.property('number3');
+                expect(obj.number3).to.eql([ 5, 12, 15, 30, 37, 48 ]);
+            });
+            it('should return array', function(){
+                var propertyOf = _.propertyOf.apply(_, params);
+                var result = _.map([['number1', '2'], ['number2', '3'], ['number3', '4']], propertyOf);
+                expect(result).to.eql([15, 30, 37]);
+            });
+        });
+    });
+    describe('range', function () {
+        describe('range1', function () {
+            var params;
+            beforeEach(function(){
+                params = datasets.range1();
+            });
+            it('should check types', function(){
+                var elem1 = params[0];
+                expect(typeof elem1).to.eql('number');
+                expect(elem1 instanceof Array).to.eql(false);
+                expect(params).to.have.length(1);
+                expect(elem1).to.equal(6);
+            });
+            it('should return array of range values', function(){
+                expect(_.range.apply(_, params)).to.eql([0,1,2,3,4,5]);
+            });
+        });
+
+        describe('range2', function () {
+            var params;
+            beforeEach(function(){
+                params = datasets.range2();
+            });
+            it('should check types', function(){
+                var elem1 = params[0],
+                    elem2 = params[1];
+                expect(elem1 instanceof Array).to.eql(false);
+                expect(typeof elem1).to.eql('number');
+                expect(typeof elem2).to.eql('number');
+                expect(params).to.have.length(2);
+            });
+            it('should return array of range values', function(){
+                expect(_.range.apply(_, params)).to.eql([5, 6, 7, 8]);
+            });
+        });
+
+        describe('range3', function () {
+            var params;
+            beforeEach(function(){
+                params = datasets.range3();
+            });
+            it('should check types', function(){
+                var elem1 = params[0],
```

Comment:
Creating new variables for every param is not good idea I think. We have good and readable access to params by Array getter.